Source/Filter/Filter.cpp

```diff
@@ -148,7 +148,7 @@ void Filter::DoFilter (const Box& tbx,
         for         (int iz=0; iz < slen_local.z; ++iz){
             for     (int iy=0; iy < slen_local.y; ++iy){
                 for (int ix=0; ix < slen_local.x; ++ix){
-                    Real sss = sx[ix]*sz[iy];
+                    Real sss = sx[ix]*sy[iy];
```

Comment:
Is this what we want? Isn't this labelled `sz` in WarpX?

Reply:
This is intended to be independent of what the coordinates are and so work for any system that is 2d. Perhaps I can change the names to use `1` and `2` instead of `x` and `y`,